sim/common/cata/damage_procs.go

```diff
@@ -39,4 +41,95 @@ func init() {
 			Callback: core.CallbackOnSpellHitDealt,
 		},
 	})
+
+	core.NewItemEffect(68925, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		procChance := 0.3 //TODO: placeholder, get real proc chance
```

Comment:
simc (latest version before MoP) has these at 15% chance, should probably start there and see if anything needs adjusting after launch.

Reply:
I will update these to simc value, I tried to find the proc chance in the db/wowhead but had no success finding it. Think this is good for now.